Valigator.Core/Extensions/DataExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Functional;
+using Valigator.Core.DataContainers.Factories;
+using Valigator.Core.DataSources;
+using Valigator.Core.StateValidators;
+using Valigator.Core.ValueValidators;
+
+namespace Valigator.Core.Extensions
+{
+	public static class DataExtensions
+	{
+		/// <summary>
+		/// Elements are Unique
+		/// </summary>
+		public static DataSourceStandardStandard<CollectionDataContainerFactory<RequiredCollectionStateValidator<TValue>, TSource, TValue>, TValue[], TValue[], ItemCountValidator<TValue>, CustomValidator<TValue[]>> ElementsUnique<TSource, TValue, TProperty>(
```

Comment:
`ElementsUnique` extension. I think it would be useful to add into Valigator now as I needed this for a test.